doc/DeveloperGuide.md

```diff
@@ -59,6 +59,9 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | edit a person | edit and update the contact details of a person
+`* * *` | user | clear the address book | delete all entries in the address book
+`* *` | user | undo and redo my actions | correct my mistakes easily
```

Comment:
You can identify the specific category of users as well when you write user stories. This allows you to define a target audience better.